src/FTPServer.jl

```diff
@@ -29,7 +29,7 @@ const PYTHON_CMD = joinpath(
 
 function __init__()
     Memento.register(LOGGER)
-
+    Conda.add("openssl==1.1.1d")
```

Comment:
Should have a comment explaining why

Reply:
Also maybe a link to an issue

Reply:
This should be added into `deps/build.jl` so it only happens when the package is installed.